### PR TITLE
Issue #479 local UDFs now use the same dimension order as in backend

### PR DESCRIPTION
**Pull request by JeroenVerstraelen**

### CODE REVIEW

**Comment by soxofaan** [comment]
This one deserves a note in the CHANGELOG I think

**Comment by JeroenVerstraelen** [comment]
> This one deserves a note in the CHANGELOG I think
Changelog entry has been added.

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Fixed dimension ordering in execute_local_udf to (t, bands, y, x). Previously, this was usually (t, bands, x, y).
```
Comment (soxofaan) [inline CHANGELOG.md:19]:
```suggestion
- Enforce `XarrayDataCube` dimension order in `execute_local_udf()` to (t, bands, y, x) to improve UDF interoperability with existing back-end implementations.```

**Comment by soxofaan** [comment]
squashed and merged in 7e3d4b9253a38b1a8adcf0af3373291863935bdb

**Pull request closed without merging**